gradle.properties

```diff
@@ -1,3 +1,3 @@
-release_version=1.2.2
+release_version=1.2.3
```

Comment:
Please, update to 1.3.0. We extending the API so we need to increase minor version